canonical-web-and-design/ubuntu.com
# Tidy up the js across the site

## Done
Tidy up the JS across the site.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run the site and click through and check it works
